lib/engines/content_block_manager/app/lib/content_block_manager/publishable.rb

```diff
@@ -21,6 +21,7 @@ def publish_with_rollback(content_block_edition)
           ],
         },
       )
+      dequeue_content_block_edition(content_block_edition) if content_block_edition.scheduled?
```

Comment:
should we dequeue all previous editions even if the current one isn't scheduled? Just noticed some weird behaviour 
* if I schedule an edition, then publish a new one
* the scheduled edition will override it when it's publish time comes.
* but it won't override the document title 🤔 
 Maybe a different PR but probably want to do asap?

Reply:
I wonder if we also need a another `state` for editions that have been dequeued, to avoid any confusion down the line, as otherwise the editions will always be in the state `scheduled` I think? Whitehall have `superseded`

Reply:
Oooh, that's a good point. Probably belongs in its own PR though. I'll make a card

Reply:
Have made a card https://trello.com/c/BPc2NY8s/755-dequeue-scheduled-editions-when-publishing-a-new-edition